static/src/javascripts/projects/common/modules/email/email.js

```diff
@@ -116,10 +97,6 @@ define([
 
             $(iframe).remove();
 
-            getOmniture().then(function (omniture) {
-                omniture.trackLinkImmediate('rtrt | email form inline | ' + analytics.formType + ' | ' + analytics.listId + ' | ' + analytics.signedIn + ' | form hidden');
```

Comment:
Does this not want to be replaced with `googleAnalytics.trackNonClickInteraction('rtrt | email form inline | ' + analytics.formType + ' | ' + analytics.listId + ' | ' + analytics.signedIn + ' | form hidden');`?